src/components/1/SelectedArticle.jsx

```diff
@@ -36,12 +41,26 @@ const SelectedArticle = ({ article_id }) => {
     .catch((err)=>{
       setErr({err})
     })
-  }, [article_id]);
+  }, []);
 
   if(error) {
     return <ErrorPage message={error.err.message}/>
   }
-  
+  const handleDelete = (comment_id, author) => {
+    if(activeUser && activeUser.username===author){
```

Comment:
Rather than handle authorisation after a user has clicked the delete button, it would be a better user experience if the delete button was only available for comments made by the active user